taxi/tests/test_models.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+class ModelTests(TestCase):
+    def test_manufacturer_str(self):
+        manufacturer = Manufacturer.objects.create(name="test",
+                                                   country="test")
+        self.assertEqual(str(manufacturer),
+                         f"{manufacturer.name} {manufacturer.country}")
+
+    def test_driver_str(self):
+        driver = get_user_model().objects.create(username="test",
+                                                 first_name="test",
+                                                 last_name="test")
+        self.assertEqual(str(driver),
+                         f"{driver.username} ({driver.first_name} "
+                         f"{driver.last_name})")
```

Comment:
The string representation of the driver should match the format specified in the model's `__str__` method. Ensure that the `__str__` method in the Driver model returns the expected format.